### ISSUE TITLE
How to deal with NDVI as input

### ISSUE THREAD

**Issue by kvantricht** [nasaharvest/presto#16]
I'm a little bit confused from the examples on how to deal with NDVI as input to Presto, using the following method:
https://github.com/nasaharvest/presto/blob/d498ba686bedafd113a0755c6d1927193769a920/presto/dataops/utils.py#L17:L28

It seems to not support NDVI as input while the current Presto encoder does. Any help with respect to how we can add NDVI would be appreciated!

**Comment by gabrieltseng** [comment]
Hi @kvantricht -

This seems like a bug in the function. I've fixed it in #18 - thanks for spotting this.

If you have your own NDVI values you would like to add, one way to do this is:

```python
from presto import construct_single_presto_input
from presto.dataops.pipelines.s1_s2_era5_srtm import NORMED_BANDS

x, mask, dw = construct_single_presto_input(...)
x[:, NORMED_BANDS.index("NDVI")] = my_ndvi_values
mask[:, NORMED_BANDS.index("NDVI")] = 0
```

**Comment by kvantricht** [comment]
Hi @gabrieltseng,

nice, thanks for fixing! I didn't realize NDVI would be automatically computed if B4 and B8 were provided, so that's convenient. I won't provide it myself then!
